src/languages/en-US.ts

```diff
@@ -1970,6 +1970,10 @@ export default class extends Language {
 		COMMAND_HELP_TITLE: (name, description) => `📃 | ***Help Message*** | __**${name}**__\n${description}\n`,
 		COMMAND_HELP_USAGE: usage => `📝 | ***Command Usage***\n\`${usage}\`\n`,
 		COMMAND_HELP_EXTENDED: extendedHelp => `🔍 | ***Extended Help***\n${extendedHelp}`,
+		COMMAND_SUPPORT_DESCRIPTION: `Show support instructions`,
+		COMMAND_SUPPORT_EXTENDED_HELP: `Literally gives you a link to our support server. What did you really expect this to do :thinking:`,
```

Comment:
```suggestion
		COMMAND_SUPPORT_EXTENDED: `Literally gives you a link to our support server. What did you really expect this to do :thinking:`,
```

This should be renamed to match the consistency of the rest of the files, and probably use the language builder as well

Reply:
resolved in d68d5aa